code/style.css

```diff
@@ -1,4 +1,276 @@
+/**************** BASE *****************/
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
 html {
-  background: #0025ff;
+  box-sizing: border-box;
+  background: #fff;
+}
+
+body {
+  background: #000;
+  color: #000;
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+h1 {
+  font-family: helvetica;
+  font-weight: 500;
+  font-size: 2.5rem;
+  text-align: center;
+  color: #fee400;
+  margin: 50px 0 20px 0;
+}
+
+h2 {
+  font-family: helvetica;
+  font-weight: 400;
+  font-size: 2rem;
+  text-align: center;
+  color: #fee400;
+  margin: 32px 0 20px 0;
+}
+
+p {
+  font-family: helvetica;
+  font-weight: 300;
+  font-size: 1rem;
+}
+
+img {
+  width: 100%;
+  height: 250px;
+  max-width: 383px;
+}
```

Comment:
Your images look a little distorted on some screen-widths, I would suggest adding `object-fit: cover;` to crop them and keep the ratio :)